lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -92,6 +92,8 @@ export const BrowserAuthErrorMessages = {
         "Invalid base64 encoded string.",
     [BrowserAuthErrorCodes.invalidPopTokenRequest]:
         "Invalid PoP token request. The request should not have both a popKid value and signPopToken set to true.",
+    [BrowserAuthErrorCodes.failedToRetry]:
```

Comment:
Is there a place where we would ever return this error to the caller instead of invalid_grant?  I'd think that we'd never want to expose this message to the caller as invalid grant would be the "correct" error for the caller.  Apologies if I missed it.

Reply:
We discussed this as part of the design. Let me sync with you offline about this.